be/src/storage/rowset/column_reader.h

```diff
@@ -173,6 +179,23 @@ class ColumnReader {
         Reader* reader;
     };
 
+    template <int Size>
+    struct SmallBitset {
```

Comment:
Could you move this util class to util/ directory?

Reply:
Give some comments for this class, for example to explain what does the Size stand for.